pygluu/kubernetes/gui/app.py

```diff
@@ -64,30 +96,26 @@ def main():
     """
     App initialization with parser to handle argument from CLI
 
-    Arguments :
-        -H --host : define hostname
-        -p --port : define port
-        -d --debug : override debug value default is False
-
     Note :
         web logs and socketio is disabled to avoid mixing with system logs
         this make easier to read system logs.
     """
     args = parse_args()
-    host = args.host
-    port = args.port
-    debug = args.debug
-
     copy_templates()
     app = create_app()
 
     app.logger.disabled = True
-    log = logging.getLogger('werkzeug')
+    logging.getLogger('werkzeug').disabled = True
     logging.getLogger('socketio').setLevel(logging.ERROR)
     logging.getLogger('engineio').setLevel(logging.ERROR)
-    log.disabled = True
 
-    app.run(host=host, port=port, debug=debug)
+    options = {
+        'bind': '%s:%s' % (args.host, args.port),
+        'worker_class': 'gevent',
+        "debug": args.debug
```

Comment:
I can't find reference to `debug` option in Gunicorn config. Can you share the reference?

Reply:
@iromli https://docs.gunicorn.org/en/0.17.0/configure.html#debugging

Reply:
@iromli i just realized it's an old doc.. :disappointed_relieved: